### PR TITLE
216 move outlier report into component

### PR DESCRIPTION
**Pull request by abrieff**
Moves the outlier report into a component - seeing if I can close out PLUTO this morning. Functionality should be unchanged

### CODE REVIEW

File: src/pluto/components/outlier_report.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            "lotarea_numfloor": f"### Table of BBLs where bldgarea/lotarea > numfloors*2",
+        }
+    
+    @property
+    def info_dict(self):
+        return {
+            "building_area_increase": "The table displays all BBLs where building area is more than doubled since previous version.",
+            "unitsres_resarea": "The table displays all BBLs where unitsres is more than 50 and resarea is greater than 0 but the ratio of resarea:unitsres is less than 300.",
+            "lotarea_numfloor": "The table displays all BBLs where the ratio of bldgarea:lotarea is more than twice numfloors.",
+        }
```
Comment (SashaWeinstein) [inline src/pluto/components/outlier_report.py:89]: No new line at end of file makes me worry that this wasn't formatted with black?

**Review by SashaWeinstein — COMMENTED** [review]
One small comment on style, running on my machine to make sure the it's functional

**Review by SashaWeinstein — COMMENTED** [review]
This works on my machine! Just the one style comment

**Comment by abrieff** [comment]
yep should be fixed

**Review by SashaWeinstein — APPROVED** [review]
Looks good to me! Feels good to not have to scroll through 600 lines anymore

**Pull request merged**